_presenters/felipe-lee.md

```diff
@@ -10,4 +10,4 @@ twitter: felipeleeg
 website: ''
 ---
 
-I’m a full stack developer at UT Austin. I self-describe as a dog person, but somehow have 3-4 cats (one comes and goes) and no dogs at the moment. Interestingly, two of my cats were born the very first day I started as a developer so they’ve grown up with my career :smiley_cat: On my spare time I like to learn about tech I don’t use at work, play video games like Elder Scrolls, and spend with my partner and friends.
+I’m a full stack developer at UT Austin. I self-describe as a dog person, but somehow have 3-4 cats (one comes and goes) and no dogs at the moment. Interestingly, two of my cats were born the very first day I started as a developer so they’ve grown up with my career. :smiley_cat: In my spare time I like to learn about tech that I don’t use at work, play video games like Elder Scrolls, and spend time with my partner and friends.
```

Comment:
Spell out UT?

It's clear to most Americans, but for international audiences, it's an unclear initialism.